editoast/src/client/redis_config.rs

```diff
@@ -0,0 +1,21 @@
+use clap::Args;
+use derivative::Derivative;
+
+#[derive(Args, Debug, Derivative, Clone)]
+#[derivative(Default)]
+pub struct RedisConfig {
+    #[derivative(Default(value = r#""localhost".into()"#))]
+    #[clap(long, env, default_value = "localhost")]
+    /// Redis database host
+    redis_host: String,
+    #[derivative(Default(value = "6379"))]
+    #[clap(long, env, default_value_t = 6379)]
+    /// Redis database port
+    redis_port: u16,
+}
+
+impl RedisConfig {
+    pub fn url(&self) -> String {
+        format!("redis://{}:{}", self.redis_host, self.redis_port,)
```

Comment:
This doesn't work because we can specify a database and password: `redis://[:PASSWORD@]HOST[:PORT][/DATABASE]`. Which is the case in production.

Two ways to fix that:
- You can add a new parameter `REDIS_DATABASE` (optional). I still need to include the password in the `REDIS_HOST`.
- You can add two new parameters `REDIS_PASSWORD` (optional) and `REDIS_DATABASE` (optional)
- You can merge `redis_host` and `redis_port` into one field `redis_url` that the user has to build himself.

Personally I prefer the second and third option.